shared/data/fingerprint-protection.js

```diff
@@ -68,6 +68,18 @@
                 'targetValue': undefined
             }
         },
+        'useragent': {
+            'userAgent': {
+                'object': 'navigator',
+                'origValue': navigator.userAgent,
+                'targetValue': `"${ua}"` // Defined in chrome-events.es6.js and injected as a variable
```

Comment:
hm, why not `"targetValue": ua`? Do we really need those `"`?

Reply:
Unfortunately, yes, though it's ugly. The eventual function it gets passed to will throw an error if a string value doesn't have quotes surrounding it, because it is part of an object (line 135)